scripts/utils/slug.js

```diff
@@ -1,15 +1,15 @@
+import crypto from 'crypto';
```

Comment:
[suggest] Minor. We can specify `node:` for clarity. How about this?

```suggestion
import crypto from 'node:crypto';
```